src/strategies/silo/SiloStrategy.sol

```diff
@@ -30,14 +33,19 @@ import {EasyMathV2} from "@silo/lib/EasyMathV2.sol";
  * @author johnnyonline
  * @notice A strategy that deposits funds into a Silo and harvests incentives.
  */
-contract SiloStrategy is BaseHealthCheck, TradeFactorySwapper {
+contract SiloStrategy is BaseHealthCheck, TradeFactorySwapper, Governance2Step {
```

Comment:
we already have governance role in BaseStrategy, no need to add new ones

Reply:
https://github.com/yearn/tokenized-strategy/blob/cf791a6f2d360e5c33866c9f0de10e83085920e9/src/TokenizedStrategy.sol#L258-L279

Reply:
So using `onlyManagement` is fine? Can't see `onlyGovernance` in the link.